app/views/shell/_top.html.erb

```diff
@@ -7,6 +7,13 @@
     <% unless internal_navigation? %>
       <meta name="last-updated" content="<%= Time.current %>">
       <meta name="user-signed-in" content="<%= user_signed_in? %>">
+      <!-- Expermiental web monetization payment pointer for micropayments -->
+      <!-- It lets readers make micropayments to websites they visit. -->
+      <!-- This is step 1: Get live in production to test for platform-wide payment pointer. -->
+      <!-- Step 2: Allow authors to set their payment pointer so they can directly monetize their content based on visitors. -->
+      <!-- Step 3: Enable further functionality based on what we learn from this experimentation and how the ecosystem evolves. -->
+      <!-- Step 3: Enable further functionality based on what we learn from this experimentation and how the ecosystem evolves. -->
```

Comment:
@benhalpern 
Shouldn't this be behind  <%#  %> instead of HTML comment so it's won't show up in our markup?

Reply:
FWIW I found it helpful when I was inspecting to see what it would take to add monetization to my own website :upside_down_face: